WebRoot/js/zimbraMail/mail/view/ZmMailMsgView.js

```diff
@@ -2678,8 +2678,8 @@ function(self, attempt) {
 ZmMailMsgView.prototype.setScrollWithIframe =
 function(val) {
 	
-	if (!this._usingIframe) { return; }
-	
+	if (!this._usingIframe && !(typeof this._usingIframe === 'undefined')) { return; }
```

Comment:
typeof this._usingIframe !== 'undefined'

Reply:
Done